benchmarks/dynamo/common.py

```diff
@@ -316,12 +318,16 @@ def timed(
             # we need the mark step to send the optimizer graph out for
             # compilation.
             xm.mark_step()
+        t_iter_1 = time.perf_counter()
+        time_total += t_iter_1 - t_iter_0
```

Comment:
maybe `t_iter_end - t_iter_begin` is better?

Reply:
Renamed as suggested.